packages/button/src/Button.js

```diff
@@ -80,7 +80,8 @@ class Button extends Component<Props> {
         'has-icon-right': !!iconRight,
         'a-button-negative': !!negative,
         'a-button-outline': !!outline,
```

Comment:
in de core branding voor v4 staat wel :

<button class="a-button a-button--transparent">Transparent button</button>
ipv a-button-transparent, klopt dit dan? of is dit nog voor v3 van de core branding?
(misschien beiden toevoegen dan om compatibiliteit met v3 en v4 te behouden?)

Reply:
Nu ben ik zelf niet meer zeker 🤔  Ik check straks nog eens

Reply:
het was `a-button--transparant` zowel in 3.x.x & 4.x.x